app/models/compute_attribute.rb

```diff
@@ -18,6 +18,10 @@ def method_missing(method, *args, &block)
     raise Foreman::Exception.new(N_('%s is an unknown attribute'), method)
   end
 
+  def respond_to?(method)
```

Comment:
Can you explain this further? `respond_to?` is called in method_missing, so this is going to cause a `stack level too deep` error:

```ruby
ComputeAttribute.new.vm_description
```